bpf/lib/nodeport.h

```diff
@@ -940,8 +941,15 @@ static __always_inline int rev_nodeport_lb6(struct __ctx_buff *ctx, int *ifindex
 	l4_off = l3_off + hdrlen;
 	csum_l4_offset_and_flags(tuple.nexthdr, &csum_off);
 
+	/* rev_nodeport_lb6 function called for all outboud traffic on host
```

Comment:
I'm confused by this comment, it says 1) rev_nordport_lb6 is called for all outbount traffic, then it says 2) IF this function is called on egress, doesn't 1) imply 2)?

Reply:
I think, rev_nodeport_lb4/6 function called on both packet path[ingress and egress], for some cases on Ingress path and for all outbound traffic on egress path.